closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -261,13 +267,44 @@ static void printErrors(void)
     fflush(stderr);
 }
 
+/*
+ * We use a 8 digit map (ABBCCDDD) to represent the version of openssl.
+ * A is the major version,
+ * BB is the minor version,
+ * CC is the revision,
+ * DDD for the letter that could be present in any version.
+ * For example, if an openssl version is in this scheme 1.2.3.d
+ * where major is 1, minor is 2, reversion is 3 and last char is d -> 4.
```

Comment:
Please use "fix" and "patch" instead of "revision" (or "reversion") and "last char" for consistency with the names in `OPENSSL_VERSION_CODE` and below.

Reply:
Edited.